src/TokenManager/Jwt.php

```diff
@@ -21,6 +23,9 @@ final class Jwt implements TokenManagerInterface
      */
     private $jwtParser;
 
+    /** @var LoggerInterface */
```

Comment:
New docblock format?

Reply:
Nope, phpstorm not using the right config